packages/twenty-front/src/modules/auth/sign-in-up/hooks/useSignInUp.tsx

```diff
@@ -102,6 +102,9 @@ export const useSignInUp = () => {
   }, [setSignInUpStep, setSignInUpMode, isMatchingLocation]);
 
   const continueWithCredentials = useCallback(() => {
+    if (!form.getValues('email')) {
```

Comment:
This shouldn't be part of this PR.

Reply:
Ok, I'll make the changes after #3147 is closed